template/tests/behat/example.local.yml

```diff
@@ -54,15 +54,15 @@ local:
         # This must be an absolute path.
         drupal_root: ${docroot}
       drush:
-        alias: '@self'
+        alias: ${drush.aliases.local}
 
 example-multisite:
   extensions:
     Behat\MinkExtension:
       javascript_session: selenium2
       # Configure the base url for your site. For multisite, ensure that this
       # corresponds to a valid domain entry in sites.php.
-      base_url: http://127.0.0.1:8889
+      base_url: ${project.local.uri}
```

Comment:
So... `uri: ${project.local.protocol}://${project.local.hostname}` sets the uri to something like http://example.dev. I'm unclear why this wouldn't work? I used this sort of pattern for my multisite install with a sites.php line like `$sites['example.dev'] = 'example.dev';`.

Can you clarify?

Along with #158, I use a local file to set the per-local environment I'm using for dev.